templates/home/profile.html

```diff
@@ -18,7 +19,7 @@
       }
       body:before {
```

Comment:
Fix background css

Reply:
should i put a new picture then or should i revert changes?

Reply:
Picture is okay but it should not repeat

This is the problem
![image](https://github.com/ContriHUB/Sheher/assets/59749539/8fbb2572-f7cf-4c6f-8f03-70f1c3232546)

Reply:
Try this

```
element-selector {
    background-image: url('your-image-url.jpg');
    background-repeat: no-repeat;
}
```

Reply:
![Screenshot 2023-10-12 010102](https://github.com/ContriHUB/Sheher/assets/92302620/b776ed2b-b202-4fe0-bf23-81aa2c3c0a14)
Fixed the background but not sure why navbar css is not loading.